package.json

```diff
@@ -25,6 +25,10 @@
     "eslint-plugin-jsx-a11y": "^2.1.0",
     "eslint-plugin-react": "^6.1.2",
     "gulp": "^3.9.1",
-    "gulp-eslint": "^3.0.1"
+    "gulp-eslint": "^3.0.1",
+    "redux-logger": "^3.0.6"
+  },
+  "dependencies": {
+    "bower": "^1.8.0"
```

Comment:
this too?

Reply:
Those are dependencies added when you install the app.